README.md

```diff
@@ -1,7 +1,15 @@
-# terraform-aws-nexus [![](https://github.com/rhythmictech/terraform-aws-nexus/workflows/pre-commit-check/badge.svg)](https://github.com/rhythmictech/terraform-aws-nexus/actions) <a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=RhythmicTech" alt="follow on Twitter"></a>
+# terraform-aws-nexus
```

Comment:
Not to nitpick here but if you're going to move the banners down, having "This does some neat things:" followed by badges followed by the neat things is a bit of a stretch for the eyes.